ui/index.html

```diff
@@ -16,1978 +16,14 @@
-        <script type="text/javascript" src="scripts/ui-custom/enableStaticNAT.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/securityRules.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/vpc.js"></script>
-        <script type="text/javascript" src="scripts/vpc.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/recurringSnapshots.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/uploadVolume.js"></script>
-        <script type="text/javascript" src="scripts/storage.js"></script>
-        <script type="text/javascript" src="scripts/templates.js"></script>
-        <script type="text/javascript" src="scripts/roles.js"></script>
-        <script type="text/javascript" src="scripts/accountsWizard.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/accountsWizard.js"></script>
-        <script type="text/javascript" src="scripts/accounts.js"></script>
-        <script type="text/javascript" src="scripts/configuration.js"></script>
-        <script type="text/javascript" src="scripts/globalSettings.js"></script>
-        <script type="text/javascript" src="scripts/zoneWizard.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/physicalResources.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/zoneWizard.js"></script>
-        <script type="text/javascript" src="scripts/system.js"></script>
-        <script type="text/javascript" src="scripts/network.js"></script>
-        <script type="text/javascript" src="scripts/domains.js"></script>
-        <script type="text/javascript" src="scripts/docs.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/projectSelect.js"></script>
-        <script type="text/javascript" src="scripts/ui-custom/saml.js"></script>
-        <script type="text/javascript" src="scripts/metrics.js"></script>
-
-        <!-- Plugin/module API -->
-        <script type="text/javascript" src="scripts/ui-custom/pluginListing.js"></script>
-        <script type="text/javascript" src="plugins/plugins.js"></script>
-        <script type="text/javascript" src="modules/modules.js"></script>
-        <script type="text/javascript" src="scripts/plugins.js"></script>
-
-        <!-- Load this script after all scripts have executed to populate data -->
-        <script type="text/javascript" src="scripts/postLoad.js"></script>
+        <p>The legacy UI has been deprecated in this version as notified in the <a href="http://docs.cloudstack.apache.org/en/4.14.0.0/releasenotes/about.html#new-user-interface-depreciation-notice-of-existing-ui">previous release</a>. The legacy UI will be <a href="http://docs.cloudstack.apache.org/en/4.15.0.0/releasenotes/about.html#primate-ga-and-legacy-ui-deprecation-and-removal-notice">removed in the next release</a>.<br/>To access the legacy UI <a href="legacy">click here</a>.</p>
```

Comment:
I wouldn't even make it a link, justr the instruction "add '/lagacy' to the url in your browser"

Reply:
Actually most users won't see this as the new UI (Primate) will get installed over the folder overwriting this index.html with its own.